app/views/user_mailer/batch_subscription_notification.html.erb

```diff
@@ -4,50 +4,51 @@
   <% @creations.each_with_index do |creation, index| %>
 
     <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
-
-    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> posted a
+    <% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+    <% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+    <% title = this_work.title.html_safe %>
+    <%
+      if creation.is_a?(Work) 
+        link = style_link(this_work.title.html_safe,work_url(this_work))
+        word_count = this_work.word_count
+      else
+        link = style_link(this_work.title.html_safe,work_chapter_url(this_work, creation))
+        chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
+      end
+    %>
     <% if creation.is_a?(Work) %>
-      <%= this_work.backdate ? "backdated" : "new" %> work:
+      <%= 
+        if this_work.backdate
+          t('.html.work_backdated', byline: byline, title: title, link: link, word_count: word_count).html_safe
+        else
+          t('.html.work_new', byline: byline, title: title, link: link, word_count: word_count).html_safe
+        end
+      %>
     <% else %>
-      new chapter of <i><b><%= work_link %></b></i> (<%= this_work.word_count %> words):
-      <% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %>
+      <%= t('.html.new_chapter_of', byline: byline, title: title, link: link, word_count: creation.word_count, chapter_title: chapter_title).html_safe %>
     <% end %>
+    <%= t('.html.collected_by', pseud: this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe).html_safe %>
 
-    <p>
-      <i>
-        <%= style_bold(creation.is_a?(Work) ? work_link :
-          style_link(chapter_title, work_chapter_url(this_work, creation))) %>
-      </i> (<%= creation.word_count%> words)
-
-      <% unless @seen[this_work.id] %>
-        <br>
-
-        by <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %>
-      <% end %>
-    </p>
-
-    <% if creation.summary.present? && creation.is_a?(Chapter) %>
-      <%= style_bold("Chapter Summary:") %> <%= style_quote(raw strip_tags(sanitize_field(creation, :summary))) %>
+    <% if !creation.summary.blank? && creation.is_a?(Chapter) %>
+      <%= t '.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
```

Comment:
There's a space missing before the `%>`.

Also, I think you might've accidentally put the plain text code in the HTML file here -- the previous HTML version had some styling and stripped tags, but didn't turn it to plain text: `<%= style_quote(raw strip_tags(sanitize_field(creation, :summary))) %>`